desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -40,6 +40,10 @@ module.exports = class BidForm extends ErrorHandlingForm
     val = @$('.max-bid').val()
     @saleArtwork.cleanBidAmount val if val
 
+  removeRegistrationFormAndBidImmediately: =>
+    @$('.bid-and-register-form').hide()
```

Comment:
I did tweak the dom a little bit to make hiding the form easier... but if that is not there [assuming the class name is not used elsewhere] it would just be a `$(empty, nothing collection).hide()` which should be fine.